customer-service/src/main/kotlin/cz/kss/proj/customerservice/entity/AdditionalInformation.kt

```diff
@@ -0,0 +1,38 @@
+package cz.kss.proj.customerservice.entity
+
+import org.springframework.data.annotation.Id
+import org.springframework.data.relational.core.mapping.Table
+
+@Table("additional_information")
+data class AdditionalInformation(
+    @Id
+    var id: Long? = null,
+    val birthDay: Int,
+    val birthMonth: Int,
+    val birthYear: Int,
+    val gender: Gender,
+){
+    companion object{
+        const val ID = "id"
+        const val BIRTH_DAY = "birth_day"
+        const val BIRTH_MONTH = "birth_month"
+        const val BIRTH_YEAR = "birth_year"
+        const val GENDER = "gender"
+    }
+}
+
+fun additionalInformationFromRow(prefix: String, row: Map<String,Any>): AdditionalInformation?{
```

Comment:
This is fine for now, but later on, I'd move this to a repository, similarly how it is done in the `CustomerRepository` in the `order-service` module.